### DISCUSSION TITLE
How do I toggle dark reader from the terminal?

### DISCUSSION THREAD

**Discussion by txtyash** [darkreader/darkreader#7198] [Q&A]
No browser in linux follows the system theme. I have to do it manually. So setting the dark reader to follow the system theme is useless.

Is there a way in which I could toggle the dark theme via a shell script? Basically from outside of the browser?

I saw the API code snippet. I don't know how to utilize it.

**Comment by bershanskiy** [comment]
The easiest way to enable/disable Dark Reader is via commands. In Chrome see `chrome://extensions/shortcuts` and in Firefox open `about:addons`, then click on the "cog" icon, then on the last element in the menu.

Regarding other points:
 > No browser in linux follows the system theme.

Could you be more specific? Which distro and DE are you using?

> Is there a way in which I could toggle the dark theme via a shell script? Basically from outside of the browser?

I don't think disabling individual addons is supported, but you can use `--disable-extensions` to disable all extensions and then re-enable the needed one at a time.

> I saw the API code snippet. I don't know how to utilize it.

Not sure what you are referring to.

**Comment by txtyash** [comment]
KDE Plasma OpenSUSE Tumbleweed

I know I can use a single click to toggle dark reader but I want everything to be effortless and seamless. When I change my colorscheme to dark then the browser colorscheme should change to dark and then dark reader would be toggled automatically but the problem is that only firefox follows the global colorscheme. Brave doesn't want to add that feature. Firefox has to be relaunched to take effect. 

Right now as a workaround I'm using a script that opens another browser window toggles the dark reader and quits. It works with the help of xdotool. It is a naive solution and not pleasant.

**Reply by Gusted** [reply]
This is something that could be tackled by #3519
